### PR TITLE
fixed numVertices in transitive closure example

### PR DESCRIPTION
**Pull request by udoklein**

### CODE REVIEW

**Comment by srowen** [comment]
I agree, though this is virtually too trivial to bother with

File: examples/src/main/python/transitive_closure.py [hunk 1]
```diff
@@ -23,7 +23,6 @@
 from pyspark import SparkContext
 
 numEdges = 200
-numVertices = 100
```
Comment (srowen) [inline examples/src/main/python/transitive_closure.py]: Wait, actually isn't the bug below? `rand.randrange(0, numEdges)` should be `rand.randrange(0, numVertices)`

Comment (udoklein) [inline examples/src/main/python/transitive_closure.py]: Yes, you are right, the bug is in the rand.randrange part.

Comment (srowen) [inline examples/src/main/python/transitive_closure.py]: OK if you update this I'll merge

**Comment by udoklein** [comment]
Fixed it as suggested.

**Comment by srowen** [comment]
@udoklein last little thing, can you update the title?

**Comment by udoklein** [comment]
@srowen Better?

**Comment by srowen** [comment]
Merged to master/1.6

**Pull request closed without merging**